### DISCUSSION TITLE
Supabase Edge Functions adapter for Remix

### DISCUSSION THREAD

**Discussion by thorwebdev** [supabase/supabase#6241] [Feature Requests]
Yesterday we launched [Supabase Edge Functions](https://supabase.com/edge-functions) but in the initial beta we're only supporting POST requests, read the background [here](https://supabase.com/blog/2022/03/31/supabase-edge-functions#can-i-host-static-sites).

We've heard from the Remix community that this is disappointing, so we'd love to hear from you all what you'd consider the ideal experience for Remix and work towards this together 💪

**Comment by quangv** [comment]
Being able to deploy to Supabase sounds like it could be useful. This way the edge function is closer to the data?

**Comment by gustavopch** [comment]
Is it currently possible to run Remix on Supabase? 🤔

**Reply by toptal-dave** [reply]
Any update here?

**Comment by nikita-starostin** [comment]
any update here?
